draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -160,58 +160,52 @@
+                </t>
+                <t hangText="Provider:">
+                  The term "provider" is used synonymously with "transit provider" (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>).
+                </t>
+                <t hangText="Customer:">
+                  An autonomous system that uses a provider for transit (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>).
+                </t>
+                <t hangText="Lateral peer:">
+                The term "lateral peer" is synonymous with "non-transit" or "peer-to-peer" relationship. (<xref target="RFC7908"/>, Section 3.2).
+                </t>
+                <t hangText="CAS:">
+                  Customer AS (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>).
+                </t>
+                <t hangText="PAS:">
+                  Provider AS (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>).
+                </t>
+                <t hangText="SPAS:">
+                  Set of Provider ASes (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="3"/>).
+                </t>
+                <t hangText="VAP:">
+                  Validated ASPA Payload (<xref target="ASPA"/>).
+                </t>
+                <t hangText="Provider+:">
+                  The term "Provider+" in the definition of the hop-check function indicates any of the three possibilities: Provider, non-transparent RS, or mutual-transit neighbor (<xref target="pair-validation"/>).
+                </t>
+           </list>
+         </t>
+      </section>
+      <section title="Peering Relationships" anchor="peering">
+        <t>
+         For path verification purposes in this document, the BGP roles an AS can have in relation to a neighbor AS are customer, provider, lateral peer, Route Server (RS), RS-client, and mutual-transit.
+         These relationships are defined in <xref target="RFC9234"/>, except mutual-transit which is a special case of the complex relationship <xref target="RFC9234"/>.
+         Mutual-transit ASes MAY export everything (both customer and non-customer routes) to each other, i.e., the customer-to-provider relationship applies in each direction.
+         All roles are configured locally.
        </t>
     </section>
 
-    <section title="Peering Relationships" anchor="peering">
-      <t>
-       For path verification purposes in this document, the BGP roles an AS can have in relation to a neighbor AS are customer, provider, lateral peer, Route Server (RS), RS-client, and complex.
-       These relationships are defined in <xref target="RFC9234"/>.
-       A special case of complex relation is mutual-transit when ASes MAY export everything (both customer and non-customer routes) to each other, i.e., customer-to-provider relationship applies in each direction.
-        All roles are defined locally.
```

Comment:
roles are defined. BGP Roles are configured

Reply:
s/All roles are configured locally./All BGP Roles are configured locally./

Operator would normally configure BGP Roles locally first and then they are confirmed with the neighbor by sending/receiving [RFC9234] BGP Role in the BGP OPEN. Even if the neighbor does not send BGP Role, the locally configured BGP Role remains.

Reply:
We should have a soft reference to BGP Roles definition. We are no insisting on configuring BGP Roles at this point. That why I moved in the consideration section. So I think 'define' is better.

Reply:
> We should have a soft reference to BGP Roles definition. We are no insisting on configuring BGP Roles at this point. That why I moved in the consideration section. So I think 'define' is better.

Isn't a BGP role required for ASPA? So to have ASPA verification one must configure the BGP role.